test/integration/students/student_unauthorized_pages_test.rb

```diff
@@ -0,0 +1,25 @@
+require "test_helper"
+
+class StudentUnauthorizedPagesTest < ActionDispatch::IntegrationTest
+  def login_student
+    FactoryGirl.create(:user, :default)
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.